src/App.js

```diff
@@ -0,0 +1,18 @@
+import { Switch, Route } from 'react-router-dom';
```

Comment:
- [ ] Please create a `container` folder. Please make a distinction between components and container folders. In containers would be all the components where you manage some state where in components folder should be components used for render only like the header.